geopandas/base.py

```diff
@@ -969,6 +969,36 @@ def total_bounds(self):
     def sindex(self):
         return self.geometry.values.sindex
 
+    def sindex_generated(self):
+        """Use the `.sindex` attribute on a GeoDataFrame or GeoSeries
```

Comment:
Use the first paragraph of the docstring for `GeometryArray` here to preserve the emphasis on the core part of this fuction `...quickly checks...`

Reply:
Donzo.